app/controllers/events_controller.rb

```diff
@@ -43,7 +43,6 @@ def new
 
   # GET /events/1/edit
   def edit
-    @event.destroy # destroy old event before creating a new one
```

Comment:
While I agree that this line needed to be deleted, does that affect our ability to edit events at all?